charts/core/values.yaml

```diff
@@ -46,7 +49,7 @@ image:
   repository: dhis2/core
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 2.36.0-tomcat-8.5.34-jre8-alpine
+  tag: 2.39.0
```

Comment:
since you are updating, should we use the latest instead? https://hub.docker.com/layers/dhis2/core/2.39.1.1/images/sha256-8f83bc57686ad5414dcdb50e1aa06abef5ee838623349775c7847b8a2bac8939?context=explore